api/backend.go

```diff
@@ -304,12 +310,12 @@ func (b *StatusBackend) registerHandlers() error {
 			return nil, err
 		}
 
-		hash, err := b.SendTransaction(ctx, txArgs)
-		if err != nil {
-			return nil, err
+		result := b.SendTransaction(txArgs, "")
```

Comment:
The password needs to be passed to the backend.SendTransaction method in the backend.RegisterHandlers method. Otherwise, none of the RPC methods will work. Is it ok to not require the password in this case and just call the transactor.SendTransaction directly with the current active account?